docs/BOOTING.md

```diff
@@ -127,6 +127,8 @@ in grub.cfg with graphical GRUB menu to get the device to boot again.
     10. `eve_install_skip_persist` - do not install persist partition onto device. May be selected from graphical GRUB menu.
     11. `eve_install_skip_rootfs` - do not install rootfs partition onto device. May be selected from graphical GRUB menu.
     12. `eve_install_skip_zfs_checks` - install zfs by skipping minimum requirement checks.
+    13. `eve_install_zfs` - always install zfs, will not install ext4 if zfs install fails.
+    14. `eve_zfs_raid_level` - Sets raid level for zfs storage. Valid values are none,mirror,raid5,raid6. Default value is none.
```

Comment:
Do we need `eve_install_zfs`? And from code I can see that we parse `eve_zfs_raid_level` only if `eve_install_zfs` defined. Why?

Reply:
You mean if eve_zfs_raid_level is set at all it means we should use zfs? That makes sense.

Reply:
"Is there also a way to run with a single install+persist disk and zfs?"

Yes it works and I tested it. I also tested raid none and multiple disks.

"In the API/UI are we naming the terms "mirror", "raid5", etc? Or raidz1" etc?

It makes sense for us to use consistent user-visible terms for these."

In UI we display raid5, raid6, raid1, so I will change mirror to raid1

Reply:
Yeah I agree we do not need additional eve_install_zfs, I will remove that.